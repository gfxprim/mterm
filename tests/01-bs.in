10 10
aa\b
