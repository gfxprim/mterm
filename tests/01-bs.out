 ----------
|a         |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
 ----------
size 10x10 cursor 0x1
